jtlibrary/python/jtlibrary/src/jtlib/segmentation.py

```diff
@@ -233,18 +226,20 @@ def separate_clumped_objects(clumps_image, min_cut_area, min_area, max_area,
         maximal circularity an object must have to be considerd a clump
     max_convexity: float
         maximal convexity an object must have to be considerd a clump
-
+    allow_trimming: boolean
+        Some cuts may create a tiny third object. If this boolean is true,
+        tertiary objects < trimming_threshold (10) pixels will be removed
     Returns
     -------
     numpy.ndarray[numpy.uint32]
         separated objects
-
     See also
     --------
     :class:`jtlib.features.Morphology`
     '''
 
     logger.info('separate clumped objects')
+    trimming_threshold = 10
```

Comment:
Shouldn't this be an input parameter? e.g., is it independent of the magnification?

Reply:
This should be independent of the magnification. It is all performed on the segmented objects and the size of tiny objects depends on the thickness of the cutting line.
Anything that is bigger than a cutting artefact would be magnification dependent, but we don't want to remove those.